cassandra-framework/src/main/java/io/mesosphere/mesos/frameworks/cassandra/Main.java

```diff
@@ -161,7 +163,7 @@ private static int _main() throws UnknownHostException {
             clock,
             httpServerBaseUri.toString(),
             new ExecutorCounter(state, 0L),
-            new PersistedCassandraClusterState(state, defaultConfigRole),
+            new PersistedCassandraClusterState(state, defaultConfigRole.build()),
```

Comment:
Here we should pass the seed count and node count rather than passing a big object that has the values pulled out of it.